### PR TITLE
[CELEBORN-1295] Add tm to Celeborn's website.

### PR DESCRIPTION
**Pull request by FMX**
### What changes were proposed in this pull request?
Add trace mark symbol.

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
Should we also change mkdoc.yml and web module?

File: docs/developers/overview.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ license: |
 
 # Celeborn Architecture
 
-This article introduces high level Apache Celeborn(Incubating) Architecture. For more detailed description of each module/process,
+This article introduces high level Apache Celeborn™(Incubating) Architecture. For more detailed description of each module/process,
```
Comment (SteNicholas) [inline docs/developers/overview.md:21]: Miss the `Apache Celeborn(Incubating) solves the problems by reorganizing shuffle data in a more efficient way` in `overview.md`.

**Comment by FMX** [comment]
> Should we also change mkdoc.yml and web module?

Celeborn website doesn't use the mkdoc.yaml from this repo. It uses the https://github.com/apache/incubator-celeborn-website repo and I have changed that mkdoc.yaml.

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks! Merging to main(v0.5.0)

**Comment by SteNicholas** [comment]
@FMX, does the logo picture in `docs/logo` directory need to add tm?

**Review by cxzl25 — APPROVED** [review]

**Comment by FMX** [comment]
> @FMX, does the logo picture in `docs/logo` directory need to add tm?

Referring to other projects, I think it is not necessary.

**Review by RexXiong — APPROVED** [review]

**Comment by RexXiong** [comment]
merge to main(v0.5.0)

**Review by SteNicholas — APPROVED** [review]

**Pull request closed without merging**